### PR TITLE
9.0 kd

### PR DESCRIPTION
**Pull request by KillerDroid96**

### CODE REVIEW

**Review by neobuddy89 — COMMENTED** [review]
What's purpose of this patch?
https://github.com/crdroidandroid/android_frameworks_base/pull/162/commits/dc66d3547c1933371532aec8825f83f49451fa63

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/FingerprintUnlockController.java [hunk 1]
```diff
@@ -278,6 +278,7 @@ public void startWakeAndUnlock(int mode) {
             case MODE_NONE:
                 break;
         }
+
```
Comment (neobuddy89) [inline packages/SystemUI/src/com/android/systemui/statusbar/phone/FingerprintUnlockController.java:281]: Please ignore such unnecessary line break changes.

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java [hunk 2]
```diff
@@ -712,7 +712,7 @@ public void setLayoutTransitionsEnabled(boolean enabled) {
     }
 
     public void setWakeAndUnlocking(boolean wakeAndUnlocking) {
-        setUseFadingAnimations(wakeAndUnlocking);
+        setUseFadingAnimations(!wakeAndUnlocking);
```
Comment (neobuddy89) [inline packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java:715]: Can you explain this part?

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/FingerprintUnlockController.java (hunk 1 above)
Comment (KillerDroid96) [reply packages/SystemUI/src/com/android/systemui/statusbar/phone/FingerprintUnlockController.java:281]: Okay

File: packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java (hunk 2 above)
Comment (KillerDroid96) [reply packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java:715]: Im not sure why that is added. It was part of resolving the conflict I believe

**Comment by KillerDroid96** [comment]
Lets go ahead and exclude dc66d35

**Pull request closed without merging**